bucket/bulk-crap-uninstaller.json

```diff
@@ -5,10 +5,6 @@
     "license": "Apache-2.0",
     "url": "https://github.com/Klocman/Bulk-Crap-Uninstaller/releases/download/v4.16/BCUninstaller_4.16_portable.zip",
     "hash": "d383996cc87a82bbb6c51f775d711c887db6e44687d848b6fb0792fde703d548",
-    "pre_install": "Copy-Item \"$persist_dir\\BCUninstaller.settings\" \"$dir\" -ErrorAction SilentlyContinue -Force",
-    "uninstaller": {
-        "script": "Copy-Item \"$dir\\BCUninstaller.settings\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force"
-    },
```

Comment:
```suggestion
    "pre_install": [
        "ensure \"$persist_dir\" | Out-Null",
        "Copy-Item \"$persist_dir\\BCUninstaller.settings\" \"$dir\" -ErrorAction SilentlyContinue"
    ],
    "uninstaller": {
        "script": "Copy-Item \"$dir\\BCUninstaller.settings\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force"
    },
```

Reply:
According to my test result, `BCUninstaller.settings` was edited while exiting the app, not overwrited, and the file link works well in this situation.

Reply:
Not it is not. FIrst start of installed application will delete the file, breaking link and create new file.

![image](https://user-images.githubusercontent.com/13260377/89097934-a8b9b080-d3e3-11ea-84cd-c5db8e06bbd4.png)

First startup:

File is deleted as soon as application started

![A](https://i.imgur.com/TWLBODP.gif)

Reply:
Please check if there is any error in setting file, or it is just blank, the app only delete setting file while it is not loaded properly.